### DISCUSSION TITLE
【help】有一些使用上的问题

### DISCUSSION THREAD

**Discussion by bruce-hooo** [ant-design/ant-design#40650] [Q&A]
新人小白入门级，已经遇到过好几次了，每次写完方法之后保存，然后页面就会报这个错误：
Unhandled Rejection (ReferenceError): ReactDOM is not [defined[]
怎么刷新也不好使，之后再过一会自己就突然好用了，这是什么问题啊要怎么改一下呢？
依赖如下，照着官网教程来的：
    "@ant-design/charts": "^1.4.2",
    "@ant-design/icons": "^4.7.0",
    "@ant-design/pro-components": "^2.3.13",
    "@umijs/route-utils": "^2.1.3",
    "antd": "^4.23.3",

**Comment by Wxh16144** [comment]
可否提供一个最小 repo?
